addons/mail/static/src/js/views/activity/activity_record.js

```diff
@@ -16,6 +16,13 @@ var ActivityRecord = KanbanRecord.extend({
     // Private
     //--------------------------------------------------------------------------
 
```

Comment:
```suggestion
custom_events: Object.assign({}, KanbanRecord.prototype.custom_events, {
    open_record: (ev) => {
         ev.data.mode = "edit";
    },
},
```

That way, no need to change anything in KanbanRecord

Reply:
Thanks. Done :+1: